CHANGELOG.md

```diff
@@ -12,6 +12,30 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ## 🛠 Maintenance
 ## 📚 Documentation -->
 
+
+# [v0.1.0-alpha.1] 2021-11-18
+
+## :rocket::waxing_crescent_moon: Public alpha release
+
+> An alpha or beta release is in volatile, active development. The release might not be feature-complete, and breaking API changes are possible between individual versions.
+
+## :sparkles: Features
+
+- JSON log output
+
+## :hammer_and_wrench: Maintenance
+
+- Reduced CI build times
+- Improved overall performance by reducing the number of clones when composing and building a graphql response
+- Logging feature flags: The two remaining features available are "otlp-grpc" (on by default) and "otpl-http"
+- Move composition benchmarks to a separate crate
+- OTLP / Tracing updates
+
+## :bug: Fixes
+
+- Reworked the tests to make sure file configuration and schema updates work as indented (using the --watch flag)
+- Fix OTLP span reporting
```

Comment:
I believe (but don't have enough context here to know) this is a user-facing improvement.  Could we try something like this format for such an improvement?:

```suggestion
- **Fix OpenTelemetry span reporting**

  The OpenTelemetry span reporting will now no longer have _the thing that used to happen and be buggy or incomplete_, and will now exhibit _the new behavior that we're proud of_.
```

Otherwise this bullet point isn't really too useful.